lib/doorkeeper-jwt.rb

```diff
@@ -40,8 +40,28 @@ def self.rsa_encryption?
       /RS\d{3}/ =~ encryption_method
     end
 
+    def self.ecdsa_encryption?
+      /ES\d{3}/ =~ encryption_method
+    end
+
     def self.rsa_key
       OpenSSL::PKey::RSA.new(Doorkeeper::JWT.configuration.secret_key)
     end
+
+    def self.ecdsa_key
+      OpenSSL::PKey::EC.new(Doorkeeper::JWT.configuration.secret_key)
+    end
+
+    def self.rsa_key_file
+      OpenSSL::PKey::RSA.new(secret_key_file_open)
+    end
+
+    def self.ecdsa_key_file
```

Comment:
`private` (on line 15) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.